services/order.py

```diff
@@ -0,0 +1,39 @@
+import datetime
+from django.db import transaction
+from django.db.models import QuerySet
+from db.models import Order, User, Ticket
+from services.movie_session import get_movie_session_by_id
+
+
+def create_order(
+        tickets: list[dict],
+        username: str,
+        date: datetime.datetime = None
```

Comment:
use typehint like: Optional[YOUR_VALUE_TYPE] = None
for example: list that contains string elements and can be None has typehint Optional[list[str]]